target/linux/mvebu/image/cortexa9.mk

```diff
@@ -448,3 +448,20 @@ define Device/synology_ds213j
     -ppp -kmod-nft-offload -dnsmasq -odhcpd-ipv6only
 endef
 TARGET_DEVICES += synology_ds213j
+
+define Device/wd_cloud-mirror-gen2
+  $(Device/NAND-128K)
+  DEVICE_VENDOR := Western Digital
+  DEVICE_MODEL := MyCloud Mirror Gen 2 (BWVZ/Grand Teton)
+  DEVICE_PACKAGES += -uboot-envtools coreutils-stty mkf2fs e2fsprogs \
+	partx-utils kmod-hwmon-drivetemp -ppp -kmod-nft-offload -dnsmasq \
+	-odhcpd-ipv6only 
```

Comment:
Why do you not want to add `odhcpd-ipv6only ` ?

Reply:
I've did fallowed DS213j `PACKAGES`  selection.  Basic network  setup for NAS device will be dhcp client .

Reply:
ok, should be fine.